linkerd/app/admin/src/server.rs

```diff
@@ -254,6 +268,26 @@ where
                 }
             }
 
+            #[cfg(feature = "pprof")]
+            "/debug/pprof/profile.pb.gz" if self.pprof.is_some() => {
+                let pprof = self.pprof.expect("profiler must be configured");
```

Comment:
nit, take it or leave it: this `expect` could really be reworded to indicate that we _just_ entered a branch that's guarded by `self.pprof.is_some()` being `true`, so it should be unreachable...